gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'rubocop', '>= 1.0', '< 2.0'
```

Comment:
-[ ] Please, as the last reviewer said the dev can't run bundle install the reason for that, is just the spelling it should be `Gemfile`, not `gemfile` please change it on some machines, the file system may be configured to be case-insensitive. This means that files with different capitalizations will be treated as the same file. In this case, Ruby may be able to locate a file named gem file (with a lowercase "g") even though the convention is to use Gemfile with a capital "G".